src/styles/components/_home.scss

```diff
@@ -0,0 +1,269 @@
+.heart-svg-bg-top {
+  position: absolute;
+  z-index: 40;
+  min-width: 100%;
+  margin-bottom: 15px;
+  bottom: -30px;
+  @include breakpoint('md') {
+    display: none;
+  }
+}
+.heart-svg-bg-bottom {
+  position: absolute;
+  z-index: 40;
+  min-width: 100%;
+  margin-bottom: 15px;
+  top: 0;
+  left: 0;
+  @include breakpoint('md') {
+    display: none;
+  }
+}
+.circle-svg {
+  position: absolute;
+  right: -95px;
+  top: -20px;
+  width: 100%;
+  @include breakpoint('md') {
+    display: none;
+  }
+}
+.butterfly1-svg {
+  position: absolute;
+  z-index: 45;
+  left: 60px;
+  top: 2px;
+  @include breakpoint('md') {
+    left: 35%;
+    top: 37px;
+    display: none;
+  }
+}
+.butterfly2-svg {
+  position: absolute;
+  z-index: 45;
+  right: 60px;
+  top: 73px;
+  @include breakpoint('md') {
+    right: 40%;
+    top: 80px;
+    display: none;
+  }
+}
+.butterflies-svg {
+  display: none;
+  @include breakpoint('md') {
+    display: block;
+    margin: 0 auto;
+  }
+}
+.home-plus-signs {
+  display: none;
+  opacity: 0.2;
+  position: absolute;
+  top: -15px;
+  right: -2px;
+  width: 90%;
+  @include breakpoint('md') {
+    display: block;
+  }
+}
+
+.home__container {
+  position: relative;
+  overflow: hidden;
+  background: linear-gradient(rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.2)),
+    url('../../assets/home/background.jpeg');
+  background-size: cover;
+  background-position-y: 100%;
+  z-index: 2;
+
+  @include breakpoint('xl') {
+    background-position-y: 55%;
+  }
+}
+.home__title {
+  margin-top: 0;
+  color: $secondary-font-color;
+  font-family: Nunito;
```

Comment:
Per previous conversations with Deric, my understanding is that we are not going to use the Nunito font, we're going full Lato. Please double check on this.

Reply:
@jwilliam706 Double checked and Nunito will be removed even though it still appears in some designs. I'll update the PR.